### PR TITLE
Update teams config

### PR DESCRIPTION
**Pull request by black-desk**
Update linglong releated teams config according to the maintainer list.

### CODE REVIEW

**Comment by black-desk** [comment]
@realwujing Is it ok to remove Bruce and sukie?

**Comment by black-desk** [comment]
ping @BLumia

File: teams.yaml
Comment (BLumia) [inline teams.yaml:617]: seetaface 这些是不是可以直接去掉了？目前 deepin-community 组织下有这些包，如果是作为上游包维护的话也不适合在 linuxdeepin 组织下进行。

**Review by BLumia — COMMENTED** [review]
> Is it ok to remove Bruce and sukie?

Should I wait for the answer before merge?

> Can I rename linglong-member to linglong?

Sure.

> Will this create a new team and leave the old one as it is?

The old one will still be here but I can manually remove the old team before merge.

**Comment by black-desk** [comment]
> > Is it ok to remove Bruce and sukie?
> 
> Should I wait for the answer before merge?

These ids are not in deepin team. I don't know why they in this file. I think we can just remove it.

File: teams.yaml
Comment (black-desk) [reply teams.yaml:617]: 这些项目本身开发就非常不规范 包如果想正确地打出来 需要对原项目进行大量改动 我觉得可以算是fork了一个版本这样.

Comment (BLumia) [reply teams.yaml:618]: 好，没问题。那么 deepin-community 组织下的 seetaface 包还需要保留吗？ cc @Zeno-sole

**Review by BLumia — APPROVED** [review]

File: teams.yaml
Comment (Zeno-sole) [reply teams.yaml:618]: @BLumia  不用保留了，准备和上游分叉的话就不用在community下维护了

Comment (BLumia) [reply teams.yaml:618]: 好。

**Pull request merged**
